ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -232,35 +214,29 @@ initInternalState NodeParams {..} = do
 
         blockFetchInterface :: BlockFetchConsensusInterface peer (Header blk) blk m
         blockFetchInterface = initBlockFetchConsensusInterface
-          (tracePrefix "ChainDB" (Nothing :: Maybe peer) tracer)
           cfg chainDB getCandidates blockFetchSize blockMatchesHeader btime
 
     return IS {..}
+  where
+    NodeParams
+      { tracers, chainDB, threadRegistry, cfg, blockFetchSize
+      , blockMatchesHeader, btime, callbacks, initState
+      } = params
```

Comment:
There's nothing wrong with doing the pattern matching like this in the function arg position directly. I usually think it's clearer. And yes it's better than using `{..}`.